editor/edit-post/sidebar/post-visibility/style.scss

```diff
@@ -2,19 +2,6 @@
 	width: 100%;
 }
 
-.editor-post-visibility__toggle.button-link {
-	text-decoration: underline;
-	color: $blue-wordpress;
-
-	&:focus {
-		outline: none;
```

Comment:
Noting that this style does not exist on the base `.button-link`, but also that it's questionable we'd want it.